core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -269,7 +269,10 @@ import scala.compat.java8.FutureConverters._
                     continue()
                   } else if (settings.eventsByTagEventualConsistency < settings.refreshInterval) {
                     // No point scheduling for EC if the QueryPoll will come in beforehand
-                    scheduleOnce(TagNotification, settings.eventsByTagEventualConsistency)
+                    // No point in scheduling more frequently than once every 10 ms
+                    scheduleOnce(
+                      TagNotification(System.currentTimeMillis() / 10),
+                      settings.eventsByTagEventualConsistency)
                   }
```

Comment:
@chbatey I noticed that the scheduled TagNotification is reset for each incoming notification message, which can result in endless delay if there is a steady pace of incoming notification messages.

Reply:
Do you want to fix that in here or shall i raise a new PR

Reply:
the diff you see here is my fix